config/clients/ruby/template/lib/openfga/client/openfga_client.rb.mustache

```diff
@@ -0,0 +1,490 @@
+    def list_stores(opts = {})
+      @api_client.list_stores(opts)
+    end
+
+    # Writes an authorization model
+    # Creates or updates an authorization model for a specific store.
+    # @param type_definitions [Array<TypeDefinition>] The type definitions for the authorization model.
+    # @param schema_version [String] The schema version for the authorization model.
+    # @param conditions [Hash] The conditions for the authorization model.
+    # @param opts [Hash] Optional parameters for the request.
+    def write_authorization_model(type_definitions:, schema_version:, conditions: nil, opts: {})
+      fail ArgumentError, "Missing the required parameter 'type_definitions'" if type_definitions.nil?
+      fail ArgumentError, "Missing the required parameter 'schema_version'" if schema_version.nil?
+
+      body = WriteAuthorizationModelRequest.new(
+        type_definitions:,
+        schema_version:,
+        conditions:
+      )
+
+      @api_client.write_authorization_model(store_id(opts), body, opts)
+    end
+
+    # Reads an authorization model
+    # Retrieves a specific authorization model by its ID from a given store.
+    # @param id [String] The ID of the authorization model to read.
+    # @param opts [Hash] Optional parameters for the request.
+    # @raise [ArgumentError] If the `store_id` or `id` is not provided.
+    # @return [ReadAuthorizationModelResponse] The response containing the authorization model details.
+    def read_authorization_model(id:, opts: {})
+      @api_client.read_authorization_model(store_id(opts), id, opts)
+    end
+
+    # Reads all authorization models
+    # Retrieves all authorization models for a specific store.
+    # @param opts [Hash] Optional parameters for the request.
+    # @raise [ArgumentError] If the `store_id` is not provided.
+    # @return [ReadAuthorizationModelsResponse] The response containing the list of authorization models.
+    def read_authorization_models(opts = {})
+      @api_client.read_authorization_models(store_id(opts), opts)
+    end
+
+
+
+    # Checks whether a specific relationship exists in the store.
+    #
+    # @param user [String] The user involved in the relationship.
+    # @param relation [String, Symbol] The relation to check (e.g., "reader", "owner").
+    # @param object [String] The object involved in the relationship.
+    # @param opts [Hash] Optional parameters for the check.
+    #   @option opts [Hash] :contextual_tuples Additional contextual tuples to include in the check.
+    #   @option opts [String] :authorization_model_id The ID of the authorization model to use for the check.
+    #   @option opts [Hash] :context Additional context for the check.
+    #
+    # @raise [ArgumentError] If any of the required parameters (`user`, `relation`, or `object`) are missing.
+    #
+    # @return [CheckResponse] The result of the check operation.
+    def check(user:, relation:, object:, contextual_tuples: nil, context: nil, opts: {})
```

Comment:
Same note as above, try to wrap the params to be body, options to avoid future breaking changes, and we try to have that same pattern consistent across all methods

https://github.com/openfga/js-sdk/blob/main/client.ts#L618-L630

(applies to all later variations)